tkp/steps/source_extraction.py

```diff
@@ -13,12 +13,12 @@
                                     'rms_max'])
 
 
-def extract_sources(image_path, extraction_params):
+def extract_sources(accessor, extraction_params):
     """
     Extract sources from an image.
 
     args:
-        image_path: path to file from which to extract sources.
+         images: a tuple of image DB object and accessor
```

Comment:
Your docstring has gotten out of sync here - we're back to feeding it just the accessor